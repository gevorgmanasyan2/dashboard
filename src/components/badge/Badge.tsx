import React, { FC } from 'react'
import { Badge as AntBadge } from 'antd'

export class BadgeProps {
  constructor(public count: number, public size: 'default' | 'small') {}
}

const Badge: FC<BadgeProps> = ({ count, size }) => {
  return <AntBadge count={count} overflowCount={99} size={size} />
}

export default Badge
