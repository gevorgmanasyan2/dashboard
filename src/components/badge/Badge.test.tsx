/* eslint-disable no-undef */
import React from 'react'
import { shallow } from 'enzyme'
import Badge, { BadgeProps } from './Badge'

it('should render Badge component', () => {
  const mockItem = new BadgeProps(5, 'small')

  expect(shallow(<Badge {...mockItem} />)).toMatchSnapshot()
})
