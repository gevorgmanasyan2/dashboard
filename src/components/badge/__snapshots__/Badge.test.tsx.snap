// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Badge component 1`] = `
<Badge
  count={5}
  overflowCount={99}
  size="small"
/>
`;
